install_config/install/advanced_install.adoc

```diff
@@ -659,6 +659,9 @@ Use the Device Mapper Storage Driver guide].
 available either locally or in at least one of the configured container image
 registries on the host machine.
 
+|`openshift_release`
+| Defines the version of OpenShift to install. Can be used in conjunction with package_version to install or upgrade to a specific release or asynchronous (z stream) release.
```

Comment:
s/OpenShift/{product-title}
s/package_version/\`package_version\`

Unless you have word that they're letting users skip versions in the upgrade stream, I'd also consider s/release or asynchronous/asynchronous